compiler/runtime/OMRCodeMetaDataManager.cpp

```diff
@@ -294,6 +295,12 @@ CodeMetaDataManager::insertMetaDataRangeInHash(
       return 1;
       }
 
+   // Don't insert zero sized ranges.
+   if (startPC == endPC)
+      {
+      return 1;
+      }
+
```

Comment:
Currently, you can insert metadata with a zero sized range, but the query logic will never find it. So you'll insert things into the table that can never be removed. I'll add a comment to this.